Quadriga/src/main/java/edu/asu/spring/quadriga/utilities/impl/NetworkUtils.java

```diff
@@ -0,0 +1,48 @@
+package edu.asu.spring.quadriga.utilities.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.network.IJsonCreator;
+import edu.asu.spring.quadriga.service.network.INetworkTransformationManager;
+import edu.asu.spring.quadriga.service.network.domain.ITransformedNetwork;
+import edu.asu.spring.quadriga.utilities.INetworkUtils;
+
+@Service
+public class NetworkUtils implements INetworkUtils {
+
+    @Autowired
+    private INetworkTransformationManager transformationManager;
+
+    @Autowired
+    private IJsonCreator jsonCreator;
+
+    private final Logger logger = LoggerFactory.getLogger(NetworkUtils.class);
+
+    @Override
+    public ResponseEntity<String> sendNetworkAsResponseEntity(String networkId) {
```

Comment:
not a good name for this method. The main purpose of this method is not to send the network as response entity, but to transform it. Also, you don't want to tightly couple the network format with its transformation. At least make it possible to decide what kind of format should be generated by this method (e.g. JSON or XML if we need this in the future).